### ISSUE TITLE
Strikethrough text doesn't work with page links

### ISSUE THREAD

**Issue by mmower** [Roam-Research/issues#4]
If you write

\~\~ [[Page Ref]] \~\~

You would expect to see

~~ [[Page Ref]] ~~

What you actually see is

\~\~ [[Page Ref]] \~\~

i.e. using a page reference stops the ~ being interpreted as a syntax marker.

**System Information:**
 - Device: MacBook Pro 15,3
 - OS: macOS 10.14.6
 - Browse: Chrome 81.0

**Comment by ryanguill** [comment]
this is a big one for me - I would love to be able to use this for scheduled tasks, where I schedule something for a future date and then if I am not able to do it on that date, strike through it and add a new date.  This would keep a reference that I tried to do it on the original date but didn't get to.

**Comment by aemt** [comment]
This glitch happens with Bold and Underline as well.

**Comment by Stvad** [comment]
Yeah, I believe only highlighting works properly with references =\

**Comment by Stvad** [comment]
this should probably merged with #7
